crates/tree_sitter_crochet/corpus/expressions.txt

```diff
@@ -315,8 +315,8 @@ Objects
 --------------------------------------------------------------------------------
 
 (program
-  (statement_block)
-  (empty_statement)
+  (expression_statement
+    (object))
```

Comment:
Given the other things in this test case are parsed as objects, I think it makes sense to parse `{};` as an object as well.